tests/pages/password.js

```diff
@@ -0,0 +1,16 @@
+import {
+  create,
+  visitable
+} from 'ember-cli-page-object';
+import form from './components/password/forgot-password';
+import resetPasswordForm from './components/password/reset-password';
+import errorFormatter from './components/error-formatter';
+
+export default create({
+  visitReset: visitable('/password/reset'),
+  visitForgot: visitable('/password/forgot'),
+
+  form,
+  resetPasswordForm,
+  errorFormatter
+});
```

Comment:
Yeah I see your point.  Not really.  Only because I have an `acceptance/password-test`.  So sounds like I should split them up?